img/cloud/vmware.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="450.513" height="450.513" viewBox="0 0 119.198 119.198"><g fill="#717175"><path d="M53.194 67.824l-5.421-15.426c-.071-.213-.174-.494-.174-.741 0-.67.553-1.304 1.345-1.304.692 0 1.142.458 1.35 1.127l4.49 13.292 4.524-13.328c.206-.633.623-1.09 1.31-1.09h.14c.727 0 1.175.457 1.381 1.09l4.489 13.328L71.19 51.41c.174-.566.55-1.058 1.277-1.058.76 0 1.313.597 1.313 1.27 0 .246-.104.526-.172.74l-5.457 15.46c-.28.808-.83 1.232-1.452 1.232h-.103c-.692 0-1.211-.424-1.454-1.231l-4.452-13.108L56.2 67.824c-.242.807-.76 1.231-1.45 1.231h-.072c-.655 0-1.206-.424-1.483-1.231M92.714 51.698c0-.744.587-1.38 1.313-1.38.763 0 1.35.6 1.35 1.38v3.13c1.314-3.04 4.228-4.475 6.159-4.475.798 0 1.318.6 1.318 1.377 0 .744-.484 1.272-1.18 1.379-3.46.424-6.296 3.076-6.296 8.346v6.197c0 .743-.554 1.378-1.315 1.378-.762 0-1.349-.603-1.349-1.378zM111.4 69.202c-4.814 0-8.747-3.816-8.747-9.413v-.073c0-5.21 3.56-9.41 8.408-9.41 5.185 0 8.137 4.37 8.137 9.167a1.24 1.24 0 01-1.222 1.262H105.3c.37 3.916 3.046 6.12 6.168 6.12 2.17 0 3.76-.874 5.05-2.099.202-.173.44-.313.747-.313.642 0 1.152.525 1.152 1.157 0 .313-.134.629-.408.87-1.66 1.645-3.627 2.732-6.61 2.732m5.187-10.463c-.27-3.29-2.104-6.158-5.595-6.158-3.051 0-5.358 2.622-5.69 6.158zM73.83 63.537v-.071c0-3.842 3.104-5.888 7.62-5.888 2.279 0 3.9.318 5.49.778v-.637c0-3.276-1.972-4.97-5.314-4.97-1.795 0-2.84.234-4.152.831a1.31 1.31 0 01-.48.107c-.621 0-1.175-.53-1.175-1.164 0-.562.24-.91.726-1.13 1.756-.778 3.08-1.037 5.356-1.037 2.517 0 4.449.667 5.757 2.008 1.209 1.233 1.831 2.996 1.831 5.322v10.057c0 .773-.552 1.338-1.28 1.338-.755 0-1.269-.565-1.269-1.269v-1.74c-1.243 1.66-3.316 3.139-6.449 3.139-3.314 0-6.662-1.938-6.662-5.674m13.144-1.41v-1.762a18.264 18.264 0 00-5.246-.778c-3.342 0-5.208 1.483-5.208 3.772v.07c0 2.29 2.07 3.63 4.486 3.63 3.274 0 5.968-2.045 5.968-4.932M4.87 51.497c-.584-1.31-2.028-1.902-3.376-1.289-1.35.613-1.846 2.142-1.236 3.452l5.66 12.57c.89 1.971 1.829 3.004 3.589 3.004 1.881 0 2.701-1.125 3.59-3.004 0 0 4.938-10.984 4.987-11.098.051-.117.21-.472.711-.47.425.005.783.35.783.814V66.22c0 1.655.898 3.014 2.627 3.014 1.728 0 2.663-1.359 2.663-3.014v-8.79c0-1.695 1.188-2.794 2.808-2.794 1.62 0 2.7 1.137 2.7 2.795v8.789c0 1.655.901 3.014 2.627 3.014 1.728 0 2.666-1.359 2.666-3.014v-8.79c0-1.695 1.184-2.794 2.806-2.794 1.618 0 2.7 1.137 2.7 2.795v8.789c0 1.655.901 3.014 2.629 3.014 1.726 0 2.663-1.359 2.663-3.014V56.216c0-3.677-2.892-6.252-6.374-6.252-3.478 0-5.656 2.458-5.656 2.458-1.157-1.533-2.753-2.455-5.454-2.455-2.85 0-5.344 2.455-5.344 2.455-1.158-1.533-3.13-2.455-4.763-2.455-2.525 0-4.53 1.135-5.755 3.993l-3.614 8.706z"/></g></svg>
```

Comment:
Why are you using both viewbox and width+height? Wouldn't `viewBox="0 0 450.513 450.513"` work better?

Reply:
This one is a bit smaller:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 180 28.48" fill="#696566"><path d="M80.33,26.39,72.14,3.6a3.93,3.93,0,0,1-.26-1.1,2,2,0,0,1,2-1.93,2.11,2.11,0,0,1,2,1.67l6.78,19.64L89.56,2.19a2.07,2.07,0,0,1,2-1.62h.21a2.13,2.13,0,0,1,2.09,1.62l6.78,19.69,6.89-19.74A2,2,0,0,1,109.44.57a1.93,1.93,0,0,1,2,1.88,3.86,3.86,0,0,1-.26,1.09l-8.24,22.85a2.37,2.37,0,0,1-2.2,1.82h-.15a2.26,2.26,0,0,1-2.2-1.82L91.65,7,84.87,26.39a2.25,2.25,0,0,1-2.19,1.82h-.11a2.38,2.38,0,0,1-2.24-1.82"/><path d="M140,2.56a2,2,0,1,1,4,0V7.19a11,11,0,0,1,9.3-6.62,1.94,1.94,0,0,1,2,2,2,2,0,0,1-1.78,2C148.31,5.27,144,9.19,144,17v9.16a2,2,0,1,1-4,0V2.56Z"/><path d="M168.23,28.43C161,28.43,155,22.79,155,14.52v-.11C155,6.71,160.39.5,167.71.5,175.54.5,180,7,180,14.05a1.86,1.86,0,0,1-1.85,1.87H159c.55,5.79,4.59,9,9.31,9a10.77,10.77,0,0,0,7.63-3.1,1.68,1.68,0,0,1,1.12-.46,1.73,1.73,0,0,1,1.75,1.71,1.69,1.69,0,0,1-.62,1.28,13.4,13.4,0,0,1-10,4M176.06,13c-.41-4.87-3.18-9.1-8.45-9.1-4.61,0-8.09,3.87-8.6,9.1Z"/><path d="M111.49,20.06V20c0-5.67,4.69-8.7,11.51-8.7a28.55,28.55,0,0,1,8.29,1.15v-.94c0-4.84-3-7.34-8-7.34A13.72,13.72,0,0,0,117,5.34a1.93,1.93,0,0,1-.72.16,1.79,1.79,0,0,1-1.78-1.72,1.67,1.67,0,0,1,1.1-1.67A17.89,17.89,0,0,1,123.68.58c3.8,0,6.72,1,8.69,3,1.83,1.82,2.77,4.42,2.77,7.86V26.28a1.91,1.91,0,0,1-1.94,2,1.83,1.83,0,0,1-1.91-1.87V23.81a11.82,11.82,0,0,1-9.74,4.64c-5,0-10.06-2.87-10.06-8.39M131.34,18v-2.6a28.33,28.33,0,0,0-7.92-1.15c-5,0-7.87,2.19-7.87,5.58v.1c0,3.38,3.12,5.36,6.78,5.36,4.94,0,9-3,9-7.29"/><path d="M7.35,2.27A3.79,3.79,0,0,0,2.26.36,3.73,3.73,0,0,0,.39,5.46L8.94,24c1.34,2.91,2.76,4.44,5.42,4.44,2.84,0,4.08-1.66,5.42-4.44l7.53-16.4A1.07,1.07,0,0,1,28.38,7a1.17,1.17,0,0,1,1.18,1.2V24c0,2.45,1.36,4.46,4,4.46s4-2,4-4.46V11A4,4,0,0,1,41.79,6.9,3.9,3.9,0,0,1,45.87,11V24c0,2.45,1.36,4.46,4,4.46s4-2,4-4.46V11A4,4,0,0,1,58.1,6.9,3.9,3.9,0,0,1,62.18,11V24c0,2.45,1.36,4.46,4,4.46s4-2,4-4.46V9.24C70.17,3.81,65.8,0,60.54,0A12.36,12.36,0,0,0,52,3.63C50.26,1.37,47.85,0,43.77,0,39.46,0,35.7,3.63,35.7,3.63A9.68,9.68,0,0,0,28.51,0c-3.82,0-6.85,1.68-8.69,5.91L14.36,18.77Z"/></svg>
```

Reply:
I just save it in _inkscape_ and run it through _svgo_... Usually I'm too lazy to optimize the files manually. I think that's a waste of time anyway. :man_shrugging: